example/src/main/res/layout/fragment_cash_receipt.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <LinearLayout
+        android:id="@+id/cart_totalCostRow"
```

Comment:
Nit: we shouldn't mix camel-case with underscore notation, let's pick one and stick with it :)

Reply:
addressed in [e313410](https://github.com/afterpay/sdk-android/pull/184/commits/e313410f0a2d4bd2cb20021979eb4781dfdc115e)